Create user based on ENV variables PUID & PGID

InB4DevOps opened the pull request:
This helps on (NAS) systems with additional ACLs and prevents disk write failures.

Use UID and GID from a host user with RW permissions on mounted folders as ENV variables PGID and PUID

You can find out UID/GID via `id <username>`

InB4DevOps committed:
Create user based on ENV variables PUID & PGID

This helps on NAS Systems with additional ACLs
Use UID and GID from a host user with RW permissions on mounted folders

InB4DevOps commented:
You might also want to change the readme

CanadaBry commented:
I have access to a synology box, so I am going to see if I can test running the docker on there. I'll merge once I've done my testing. Thanks!

InB4DevOps commented:
I made this so I could run the server on my Synology NAS. I for one can tell that it works but please do another test before merging.

CanadaBry committed:
Create Dockerfile

Updated Dockerfile to reflect changes on master.

Removed PUID and PGID control from Dockerfile. Will handle exclusively in server.sh through environment variables sent to container.

CanadaBry committed:
Update server.sh

Added default variable for PUID/PGID
